e2e/test/scenarios/embedding/embed-resource-downloads.cy.spec.ts

```diff
@@ -7,6 +7,7 @@ import {
 import { createMockParameter } from "metabase-types/api/mocks";
 
 const { PRODUCTS, PRODUCTS_ID, PEOPLE } = SAMPLE_DATABASE;
+import * as DateFilter from "../native-filters/helpers/e2e-date-filter-helpers";
```

Comment:
I copied the whole file from https://github.com/metabase/metabase/pull/59165 (52 backport) as a base, and adjust accordingly.